From 4f4ac2f7829bf4efb6d12bd489d4cc6ac1b9458e Mon Sep 17 00:00:00 2001
From: Jason Wang <jasowang@redhat.com>
Date: Wed, 12 Nov 2014 07:37:51 -0500
Subject: [virt] hyperv/vmbus: Increase the limit on the number of pfns we can handle

Message-id: <1415777871-62253-1-git-send-email-jasowang@redhat.com>
Patchwork-id: 99843
O-Subject: [RHEL7.1 PATCH] Drivers: hv: vmbus: Increase the limit on the number of pfns we can handle
Bugzilla: 1160130
RH-Acked-by: Radim Krcmar <rkrcmar@redhat.com>
RH-Acked-by: Vitaly Kuznetsov <vkuznets@redhat.com>
RH-Acked-by: Amos Kong <akong@redhat.com>

From: K. Y. Srinivasan <kys@microsoft.com>

Bugzilla: 1160130
Brew Build: https://brewweb.devel.redhat.com/taskinfo?taskID=8193181
Test status: Tested by QE

Increase the number of PFNs we can handle in a single vmbus packet.
Some network packets may have more PFNs than the current limit we have.
This is not a bug and this patch can be applied to the *next tree.

Signed-off-by: K. Y. Srinivasan <kys@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
(cherry picked from commit 7e5ec36834a5752304dc0751dfab42ba6ad64a85)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/include/linux/hyperv.h b/include/linux/hyperv.h
index f865075..9e4f507 100644
--- a/include/linux/hyperv.h
+++ b/include/linux/hyperv.h
@@ -37,7 +37,7 @@
 #include <linux/mod_devicetable.h>
 
 
-#define MAX_PAGE_BUFFER_COUNT    19
+#define MAX_PAGE_BUFFER_COUNT    32
 #define MAX_MULTIPAGE_BUFFER_COUNT   32 /* 128K */
 
 #pragma pack(push, 1)
-- 
1.7.1